Purchases/Public/RCPurchases.m

```diff
@@ -573,17 +574,27 @@ - (void)        purchaseProduct:(SKProduct *)product
     [self.storeKitWrapper addPayment:[payment copy]];
 }
 
+- (void)restoreTransactionsProgrammaticallyWithCompletionBlock:(nullable RCReceivePurchaserInfoBlock)completion {
+    [self restoreTransactionsWithReceiptRefreshPolicy:RCReceiptRefreshPolicyNever completionBlock:completion];
```

Comment:
moved from a boolean to this enum, I think it makes the intent pretty clear

Reply:
I love it